packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -12,7 +12,7 @@ RUN dnf -y install dnf-plugins-core && \
     dnf config-manager --set-enabled PowerTools
 
 RUN dnf copr enable nmstate/ovs-el8 -y && \
-    dnf copr enable networkmanager/NetworkManager-1.20 -y
+    dnf copr enable networkmanager/NetworkManager-1.22 -y
```

Comment:
This repository does not exist. The repository should be `networkmanager/NetworkManager-1.22-git`. Maybe we should ask for a stable NM 1.22 copr repository.

Reply:
Yes, we need to ask for a stable copr on NM 1.22.

Reply:
> Just a quick note, none of the coprs are tracking stable releases. I only changed the naming pattern with 1.22 to make this more obvious. Edward Haas <notifications@github.com> schrieb am Di., 28. Jan. 2020, 09:28:
> […](#)
> *@EdDev* commented on this pull request. ------------------------------ In packaging/Dockerfile.centos8-nmstate-dev <[#711 (comment)](https://github.com/nmstate/nmstate/pull/711#discussion_r371661515)>: > @@ -12,7 +12,7 @@ RUN dnf -y install dnf-plugins-core && \ dnf config-manager --set-enabled PowerTools RUN dnf copr enable nmstate/ovs-el8 -y && \ - dnf copr enable networkmanager/NetworkManager-1.20 -y + dnf copr enable networkmanager/NetworkManager-1.22 -y Yes, we need to ask for a stable copr on NM 1.22. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <#711?email_source=notifications&email_token=AABQIWCUDORBYGOEZ3MJVFLQ77UDFA5CNFSM4KMCR462YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCTIESYQ#discussion_r371661515>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AABQIWGS2ENRYL5O6WE73ADQ77UDFANCNFSM4KMCR46Q> .

Thanks Till! @EdDev I think you can update the PR now :-)